assets/apps/components/src/Controls/FormTokenFieldControl.js

```diff
@@ -26,21 +43,34 @@ const FormTokenFieldControl = ({
 
 	const getValueKeys = () => {
 		// Convert option ids into labels
-		const valueKeys = [];
-		for (const optionId of value) {
-			const wantedPost = choices.find((option) => {
-				return option.id === optionId;
-			});
-			if (wantedPost === undefined || !wantedPost) {
-				continue;
+
+		// Keep compatibility with previous version
+		if (
+			suggestions.length !== 0 &&
+			value.length > 0 &&
+			typeof value[0] !== 'object'
+		) {
+			const valueKeys = [];
+			for (const optionId of value) {
+				const wantedPost = suggestions.find((option) => {
+					return option.id === optionId;
+				});
+				if (wantedPost === undefined || !wantedPost) {
```

Comment:
`!wantedPost` is true if `wantedPost` is `undefined`.
```suggestion
				if (!wantedPost) {
```

should be enough.